tensilelite/Tensile/Tests/common/gemm/fp8_SVW_16_Bias_S_scaleAlphaVector_bpl64.yaml

```diff
@@ -0,0 +1,80 @@
+TestParameters:
+  marks: [skip-gfx900, skip-gfx906, skip-gfx908, skip-gfx90a, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030, skip-gfx1100, skip-gfx1101, skip-gfx1102, skip-gfx1200, skip-gfx1201] # not supported by arch
+
```

Comment:
Please add testing into existing testing yaml.
ex: fp8.yaml

Reply:
@jichangjichang - Thanks for comment, I removed the new file and added test to the fp8.yaml (I made sure it is getting `AssertionError: bad bpl` without change and is passing with the changes